### PR TITLE
Resume post

### PR DESCRIPTION
**Pull request by marcusmoncayo**
This is the resume post/create api call.

### CODE REVIEW

**Comment by r2d2CB** [comment]
&nbsp;<b>Code Climate</b> has <a href="https://codeclimate.com/repos/54242d7ce30ba04b220066b1/compare/resume_post">analyzed this pull request</a>.

File: spec/cb/utils/response_map_spec.rb [hunk 1]
```diff
@@ -18,15 +18,15 @@ module Cb
         request_namespace = Cb::Requests::EmailSubscription
         response_namespace = Cb::Responses::EmailSubscription
 
-        expect(response_map.response_for(request_namespace::Retrieve)).to eq response_namespace::Response
-        expect(response_map.response_for(request_namespace::Modify)).to eq response_namespace::Response
+        expect(response_map.response_for(request_namespace::Retrieve)).to eq(response_namespace::Response)
```
Comment (David-Sharpe) [inline spec/cb/utils/response_map_spec.rb]: I'm guessing the inclusion of parentheses is a boy scouting effort?

**Comment by David-Sharpe** [comment]
This PR looks good to me.  Curious what someone else has to say as this is my first ruby-cb-api gem PR.

File: lib/cb/requests/resumes/post.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          {
+            'HostSite' => Cb.configuration.host_site,
+            'Content-Type' => 'application/json',
+            'Authorization' => three_scale_bearer_token
+          }
+        end
+        
+        def three_scale_bearer_token
+          "Bearer #{args[:three_scale_token]}"
+        end
```
Comment (cblant20) [inline lib/cb/requests/resumes/post.rb]: could you explain this? I don't yet understand the 3 scale stuff.

File: spec/cb/utils/response_map_spec.rb (hunk 1 above)
Comment (helious) [inline spec/cb/utils/response_map_spec.rb]: seems that way

**Comment by cblant20** [comment]
I am happy with merging this, but I'd like another set of eyes. :)

File: lib/cb/requests/resumes/post.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            privacySetting: args[:privacy_setting],
+            resumeFileData: args[:resume_file_data],
+            resumeFileName: args[:resume_file_name],
+            hostSite: args[:host_site]
+          }.to_json
+        end
+
+        def headers
+          {
+            'HostSite' => Cb.configuration.host_site,
```
Comment (helious) [inline lib/cb/requests/resumes/post.rb]: do we need to pass this header along?

File: lib/cb/requests/resumes/post.rb (hunk 2 above)
Comment (helious) [inline lib/cb/requests/resumes/post.rb]: this is what is passed in the headers to the API so the API can get the user did

**Comment by David-Sharpe** [comment]
The hostSite header is not used in this API so @helious's comment is a valid question.  However, doing this breaks our API convention.  All other host site specific API's take HostSite in the header.  I thought we should leave it in the gem for now, update the API, then minor version bump the gem.  However, I think we might need to have the discussion of how to handle Hostsite.

**Comment by marcusmoncayo** [comment]
@David-Sharpe added header back to stick with convention

File: lib/cb/requests/resumes/post.rb (hunk 2 above)
Comment (marcusmoncayo) [inline lib/cb/requests/resumes/post.rb]: the header that is sent will look like this
Authorization: 'Bearer {token}'

{token} will be passed to the api request. It is retrieved by consumer auth and passed into the api request by consumer-resume. you will see this in my next pr.

**Pull request merged**